Update ports.mdx (#16039)
hashicorp/consul: website/content/docs/install/ports.mdx

| HTTPS: The HTTPs API                                                                                          | disabled (8501)\* |
| gRPC: The gRPC API                                                                                            | disabled (8502)\* |
| gRPC: The gRPCs API                                                                                           | 8403              |
| LAN Serf: The Serf LAN port (TCP and UDP)                                                                     | 8301              |
| Wan Serf: The Serf WAN port (TCP and UDP)                                                                     | 8302              |
| server: Server RPC address (TCP Only)                                                                         | 8300              |
| Sidecar Proxy Min: Inclusive min port number to use for automatically assigned sidecar service registrations. | 21000             |
| Sidecar Proxy Max: Inclusive max port number to use for automatically assigned sidecar service registrations. | 21255             |

> \*For `HTTPS` and `gRPC(s)` the ports specified in the table

Reviewer: ```suggestion
\*For `HTTPS` and `gRPC(s)` the ports specified in the table
```
What is there a `(s)` after `gRPC`? Do we mean that the ports for HTTP, gRPC, and the gRPC API are recommendations? Or do we mean `gRPC` and `gRPCs`?

Author: The trailing s should indicate that you can (or indeed should) use TLS for your grpc connections. We can omit it to clarify the documentation

**gRPC API** (Optional). Currently gRPC is
only used to expose the xDS API to Envoy proxies. It is off by default, but port 8502 is a convention used by various tools as the default. Defaults to 8502 in `-dev` mode.

**gRPCs API**. Currently gRPC over TLS is

> only used to expose the xDS API to Envoy proxies. It is [on by default](https://developer.hashicorp.com/consul/docs/agent/config/config-files#grpc_tls_port), and we recommend using 8503 for grpc_tls as your conventional gRPC port number, as it allows some tools to work automatically. 

Reviewer 2: I think port 8503 is also used for peering.

> only used to expose the xDS API to Envoy proxies. It is [on by default](https://developer.hashicorp.com/consul/docs/agent/config/config-files#grpc_tls_port), and we recommend using 8503 for grpc_tls as your conventional gRPC port number, as it allows some tools to work automatically. 

Reviewer 3: ```suggestion
only used to expose the xDS API to Envoy proxies. It is [on by default](/consul/docs/agent/config/config-files#grpc_tls_port), and we recommend using 8503 for grpc_tls as your conventional gRPC port number, as it allows some tools to work automatically. 
```
Hello! 👋 Quick note that this link should be relative.

> **gRPCs API**. Currently gRPC over TLS is
> only used to expose the xDS API to Envoy proxies. It is [on by default](https://developer.hashicorp.com/consul/docs/agent/config/config-files#grpc_tls_port), and we recommend using 8503 for grpc_tls as your conventional gRPC port number, as it allows some tools to work automatically. 

Reviewer: ```suggestion
**gRPCs API**. By default, Consul uses gRPC over TLS to expose the xDS API to Envoy proxies.  We recommend using port `8503` for gRPC because it allows some tools to work automatically. Refer to [`grpc_tls_port`](/consul/docs/agent/config/config-files#grpc_tls_port) for additional information.
```
Switch to active voice
